commands/lpop.md

```diff
@@ -12,7 +12,7 @@ When called without the `count` argument:
 
 When called with the `count` argument:
 
-@array-reply: the values of the first elements, or `nil` when `key` does not exist.
+@array-reply: list of popped elements, or `nil` when `key` does not exist.
```

Comment:
Please also add the example for `lpop list 2` from your PR comment. (Just like for rpop.)